src/locales/uk/parts/common.ts

```diff
@@ -28,6 +28,7 @@ export const common = {
   create: 'Створити',
   add: 'Додати',
   error404: 'Помилка 404',
+  error404Message:'На жаль, таку сторінку не знайдено.',
```

Comment:
Is there a need for 2 different 404 messages?

I suggest discussing it with BA's.

I the answer is yes - better to rename them, because currently it may cause confusion.

Reply:
Vitaly Peresada said that it will better

Reply:
But then let's use it for all 404 errors and remove line 30?

Reply:
error 404 using for title